lib/demos/material/sliders_demo.dart

```diff
@@ -321,10 +320,10 @@ class _CustomRangeThumbShape extends RangeSliderThumbShape {
     );
 
     final size = _thumbSize * sizeTween.evaluate(enableAnimation);
-    Path thumbPath;
```

Comment:
why late?

Reply:
the other way to do that was to make it nullable and force unwrap while passing to drawPath,

Reply:
I'm pretty sure it should work without late but lmk

Reply:
Yes, I see it now, I was basically using an incorrect migration for this switch case, which was leaving a couple of scenarios for it being null, since I changed that, it can be used without late as well